src/components/Header.vue

```diff
@@ -1,32 +1,73 @@
 <script lang="ts" setup>
+import { useRoute } from 'vue-router'
+
 import Logo from './shared/JomonLogo.vue'
 import UserIcon from './shared/UserIcon.vue'
 import type { User } from '/@/lib/apis'
 
 interface Props {
   me: User
 }
+const route = useRoute()
+
 defineProps<Props>()
 </script>
 
 <template>
-  <header class="flex h-12 shadow w-full items-center">
+  <header class="flex h-12 shadow w-full items-center fixed">
```

Comment:
ヘッダーにかぶったときにしたのが見えちゃってたのでこうすると良さそうです
```suggestion
  <header class="flex h-12 shadow w-full items-center fixed bg-white z-9999">
```